audio/AudioReg.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    },
+    "accelerator": "GPU",
+    "gpuClass": "standard"
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": 36,
+      "metadata": {
+        "id": "KDEvSUef1aZe"
+      },
+      "outputs": [],
+      "source": [
```

Comment:
Add the Title, your name and a short intro on your project. 

Check: [Example notebook]( https://github.com/TensorFlow-User-Group-Bangalore/Learn-Tensorflow/blob/main/computer_vision/examples/googleNet.ipynb)